validator/src/main/java/com/amazon/aoc/validators/ValidatorFactory.java

```diff
@@ -67,7 +67,7 @@ public IValidator launchValidator(ValidationConfig validationConfig) throws Exce
         expectedData = validationConfig.getExpectedMetricTemplate();
         break;
       case "eks-container-insight-logs":
```

Comment:
Can you change the name to be something like "eks-container-insight-prometheus-log" so that it's different from the container insight case? Same for line 65

Reply:
Done